Reflections/Week_4_Reflection.md

```diff
@@ -0,0 +1,13 @@
+**Will Pratt**
+1. What have you accomplished this week? (Please list each accomplishment, and explain)
+* I have drafted a new intro for my story. Using draw.io I have made a map that will show the layout for my rooms. During combat I added and revised the text to 
+  make it more interesting. I also incorporated the time library to give combat more a flow so everything doesn't happen at once and was another factor that was 
+  to make combat feel more interesting.
+2. What challenges or difficulties did you face? If you solved them, how? If not, what have you learned so far? Have you sought help or other resources?
+*  The main challenge that I had was time management and balancing my workload. I'm going to try and find a balance of how and when I should try to focus my attention
+   on work between my classes.
+3. What do you plan to accomplish in the following week? (Please list at least 3 concrete "S.M.A.R.T." goals, along with estimated number of hours to complete, or day to complete)
+* I want add more rooms which could take 1-2 days. I want to add more enemies which could take 1-2 days. I think I want to refactor my character and fight class so 
+  writing code for it should be more efficient for me moving on which could take 1-3 days.
```

Comment:
Good SMART goals. I'll be keeping an eye out for a larger graph of rooms in the near future!